src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementService.java

```diff
@@ -25,8 +25,13 @@
 
     void sendResponse(String json);
 
-    void sendRequestToHmiAndQueue(Long hearingId, HearingRequest hearingRequest, String messageType);
+    void sendRequestToHmiAndQueue(CreateHearingRequest hearingRequest, Long hearingId, String messageType);
+
+    void sendRequestToHmiAndQueue(UpdateHearingRequest hearingRequest, Long hearingId, String messageType);
```

Comment:
Hopefully based on the other comments we can undo these changes to keep it more re-usable